docs/assets/css/fluid.css

```diff
@@ -237,9 +237,14 @@ where bootstrap caption color is too light for white background */
 }
 
 /* fixing https://github.com/microsoft/FluidFramework/issues/3601
- Long words such as function names would not break in api docs causing layout overflow
+ * Long words such as function names would not break in api docs causing layout overflow.
+ * The original use-case was targeting the API member names specifically, which are generally in the
+ * first column of the API docs tables. If there are no contents for any of the items in a column,
```

Comment:
I don't think we need the note about the second column, since we're now targeting just the first one. But otherwise these docs look great! Thanks!